### DISCUSSION TITLE
Change the video orientation to portrait

### DISCUSSION THREAD

**Discussion by 8JupiterMoll8** [raspberrypi/picamera2#699] [Q&A]
Hi ,
is there away to  change the video orientation so that is encoded in portrait  using picamera2?
I want to display the videos vertically in smartphones?
Thx

**Comment by davidplowman** [comment]
I'm afraid there's no way to change the orientation directly to portrait. However, if you save it as an mp4 file then you can change the rotation metadata in the file header (using a tool like `exiftool`, for example). Many devices do it like this, and many players respect the orientation flag correctly (for example, `vlc` and `ffplay` both do).

**Comment by 8JupiterMoll8** [comment]
Thx. 
I will try exiftool. And report my expereince with it.

**Comment by sepp89117** [comment]
@8JupiterMoll8 
If you use FfmpegOutput you can rotate the output 90° clockwise with "-metadata:sv0 rotate=90". To rotate counterclockwise use rotate=-90
